asm_to_pil/src/lib.rs

```diff
@@ -198,9 +253,34 @@ impl<T: FieldElement> ASMPILConverter<T> {
             ));
         }
 
-        ConversionOutput {
-            pil: self.pil,
+        let functions: Vec<_> = if self.pc_name.is_none() {
+            input
+                .functions
+                .into_iter()
+                .map(|f| Function {
+                    name: f.name,
+                    id: f.id.unwrap(),
+                    params: f.params.clone(),
+                })
+                .collect()
+        } else {
+            // we do not expose functions for VMs
```

Comment:
not sure what this means?

Reply:
This was when objects ad their entry points, which was nothing for VMs as we cannot yet call them from the outside

Reply:
Removed now (I had missed this file when pushing)